tests/nuodb_cursor_test.py

```diff
@@ -125,6 +127,29 @@ def test_executemany_somefail(self):
 
         cursor.execute("DROP TABLE executemany_table")
 
+    def test_result_set_gets_closed(self):
+        current_version = getenv('NUODB_VERSION', None) # skipif <2.1
+        if current_version is not None and not LooseVersion(current_version) < LooseVersion("2.1"):
+        # Server will throw error after 1000 open result sets
+            con = self._connect()
+            for j in [False, True]:
```

Comment:
Have these changes been pushed to PIP?

Duke Adamonis
Director, Customer Solutions

*www.nuodb.com <http://www.nuodb.com>*

e:
*dadamonis@nuodb.com <dadamonis@nuodb.com>*p: 617-500-0001
t: @nuodbsupport

The information contained in this email is confidential and proprietary and
intended for the sole use of the intended recipient. If the reader of this
message is not the intended recipient, you are hereby notified that any
dissemination, distribution or copying of this communication is strictly
prohibited. If you have received this email in error, please notify the
sender and delete all copies.

On Tue, Aug 25, 2015 at 9:53 AM, mjmichaels <notifications@github.com>
wrote:

> In tests/nuodb_cursor_test.py
> <https://github.com/nuodb/nuodb-python/pull/119#discussion_r37866931>:
>
> > @@ -125,6 +127,29 @@ def test_executemany_somefail(self):
> >
> >          cursor.execute("DROP TABLE executemany_table")
> >
> > +    def test_result_set_gets_closed(self):
> > +        current_version = getenv('NUODB_VERSION', None) # skipif <2.1
> > +        if current_version is not None and not LooseVersion(current_version) < LooseVersion("2.1"):
> > +        # Server will throw error after 1000 open result sets
> > +            con = self._connect()
> > +            for j in [False, True]:
>
> The error message about exceeding max result sets is not a new one on the
> server side, which implies to me that maybe we were sending a
> CloseResultSet request to the server from the python driver. I would just
> like to know better what was happening before.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nuodb/nuodb-python/pull/119/files#r37866931>.
>